docs/docs/0.7.1/documentation/networking.md

```diff
@@ -188,3 +188,79 @@ Kuma DNS allocates a VIP for every Service withing a mesh. Then, it creates outb
      }
     },
 ```
+
+## Transparent Proxying
+
+There are two ways of how the service can interact with its sidecar to connect to other services.
+One is explicitly defining outbounds in the Dataplane:
+```yaml
+type: Dataplane
+...
+networking:
+  ...
+  outbound:
+  - port: 10000
+    tags:
+      kuma.io/service: backend
+```
+This approach has the disadvantage that you need to reconfigure the service to use `http://localhost:10000` when it wants to connect with service `backend`.
```

Comment:
Explain that this is what we use in Universal.

Reply:
added